ostelco-core/Models/ProductModel.swift

```diff
@@ -32,10 +32,10 @@ public struct ProductModel: Codable {
     public let properties: [String: String]
 
     public var type: String {
-        if let productType = properties["productType"] {
-            return productType.lowercased()
+        if let productClass = properties["productClass"] {
+            return productClass.lowercased()
         }
-        // default type is offer
-        return "offer"
+        // default type is simple_data
+        return "simple_data"
```

Comment:
Is it worth using `Codable` for parsing the `Properties` into some kind of object where you can do this in a typed fashion instead of using strings? Basically something where `Properties` would have a `ProductClass` that would take a value from an enum? That way you could compare that rather than raw strings.

Reply:
More changes are expected in the backend around this. It's not worth spending time until that is finalized.

Reply:
👍Fair enough